var/spack/repos/builtin/packages/snakemake/package.py

```diff
@@ -24,6 +26,11 @@ class Snakemake(PythonPackage):
 
     depends_on('python@3.3:3.6', when='@:5')
```

Comment:
Looks like 3.7+ is required now

Reply:
Ah right. I'll add that in the '@7' part